src/functionalTest/java/uk/gov/hmcts/dts/fact/admin/AdminCourtContactEndpointTest.java

```diff
@@ -0,0 +1,243 @@
+package uk.gov.hmcts.dts.fact.admin;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import uk.gov.hmcts.dts.fact.model.Court;
+import uk.gov.hmcts.dts.fact.model.admin.Contact;
+import uk.gov.hmcts.dts.fact.model.admin.ContactType;
+import uk.gov.hmcts.dts.fact.util.AdminFunctionalTestBase;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static io.restassured.RestAssured.given;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.http.HttpHeaders.*;
+import static org.springframework.http.HttpStatus.*;
+import static uk.gov.hmcts.dts.fact.util.TestUtil.*;
+
+@ExtendWith(SpringExtension.class)
+public class AdminCourtContactEndpointTest extends AdminFunctionalTestBase {
+    private static final String CONTACTS_PATH = "/" + "contacts";
+    private static final String CONTACT_TYPES_PATH = "contactTypes";
+    private static final String BIRMINGHAM_CIVIL_AND_FAMILY_JUSTICE_CENTRE_SLUG = "birmingham-civil-and-family-justice-centre";
+    private static final int ENQUIRIES_CONTACT_TYPE_ID = 22;
+    private static final String ENQUIRIES_CONTACT_TYPE = "Enquiries";
+    private static final String ENQUIRIES_CONTACT_TYPE_WELSH = "Ymholiadau";
+    private static final String TEST_NUMBER = "test number";
+    private static final Contact TEST_CONTACT = new Contact(ENQUIRIES_CONTACT_TYPE_ID, TEST_NUMBER, "explanation", "explanation cy");
+
+    @Test
+    public void shouldRetrieveContacts() {
+        final var response = given()
```

Comment:
I think you made a method below which might have the same code as here. 

You could reuse the `getCurrentContacts` method?